### PR TITLE
feat(webpack-dashboard): add new app webpack-dashboard@1.0.0

### PR DESCRIPTION
**Pull request by pascaliske**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/webpack-dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'webpack-dashboard' do
+  version '0.0.1-beta.1,1.0.0'
+  sha256 :no_check
+
+  url "https://github.com/FormidableLabs/electron-webpack-dashboard/releases/download/#{version.before_comma}/Webpack.Dashboard-#{version.after_comma}.dmg"
+  name 'Webpack Dashboard'
```
Comment (vitorgalvao) [inline Casks/webpack-dashboard.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/webpack-dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'webpack-dashboard' do
+  version '0.0.1-beta.1,1.0.0'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/webpack-dashboard.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/webpack-dashboard.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+  url "https://github.com/FormidableLabs/electron-webpack-dashboard/releases/download/#{version.before_comma}/Webpack.Dashboard-#{version.after_comma}.dmg"
+  name 'Webpack Dashboard'
+  homepage 'https://github.com/FormidableLabs/electron-webpack-dashboard'
+
+  app 'Webpack Dashboard.app'
```
Comment (vitorgalvao) [inline Casks/webpack-dashboard.rb]: Please add a [`zap`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by pascaliske** [comment]
you guys rock! 🤘🏻
just wanted to fulfill your annotations but saw that their already fixed!
thanks for merging and keep up your great work!

**Pull request merged**